src/main/java/com/ea/restaurant/grpc/exceptionHandler/GrpcExceptionAdvice.java

```diff
@@ -0,0 +1,13 @@
+package com.ea.restaurant.grpc.exceptionHandler;
+
+import io.grpc.Status;
+import net.devh.boot.grpc.server.advice.GrpcAdvice;
+import net.devh.boot.grpc.server.advice.GrpcExceptionHandler;
+
+@GrpcAdvice
+public class GrpcExceptionAdvice {
+    @GrpcExceptionHandler
+    public Status handlePermissionDenied(RuntimeException e) {
```

Comment:
This is really weird, that mean that all exceptions will be a `PERMISSION_DENIED` exception? that looks wrong.